overturemaps/cli.py

```diff
@@ -27,17 +27,26 @@ def get_writer(output_format, path, schema):
     elif output_format == "geojsonseq":
         writer = GeoJSONSeqWriter(path)
     elif output_format == "geoparquet":
-        # Update the geoparquet metadata to remove the file-level bbox which
-        # will no longer apply to this file. Since we cannot write the field at
-        # the end, just remove it as it's optional. Let the per-row bounding
-        # boxes do all the work.
+        # don't remove per row bbox and add covering spec for spatial filters
         metadata = schema.metadata
         geo = json.loads(metadata[b"geo"])
-        for column in geo["columns"].values():
-            column.pop("bbox")
+        # iter geometry columns and associated metadata, see:
+        # https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#column-metadata
+        for geom_col_name, geom_col_vals in geo["columns"].items():
+            # if the associated geom column has bbox info...
+            if "bbox" in geom_col_vals:
```

Comment:
Remove this check. If anything check that there's a bbox column in the top-level schema columns, not the geo metadata.